### ISSUE TITLE
html "iframes youtube" not rendered / html anchors not working

### ISSUE THREAD

**Issue by marseye** [Plume-org/Plume#234]
Hi, 

Writing an article in md format, there are still some limitations that I've been attemping to circumvent by inserting proper html tags, like in a youtube html embed code, such
'<iframe width="560" height="315" src="https://www.youtube.com/embed/hIMMYJ97BjM?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>'
won't let the player be displayed as intended.

A simple html anchor tag didin't work either 
' <a id="myAnchor"> <a>    
[My Anchor](#anchor)'

Plume v0.2.0 soteria.mastodon.host (Gled's place)

**Comment by fdb-hiroshima** [comment]
You will be able do use `id`s when your instance upgrade to fab6360, however depending on the Content Security Policy of the web server, `iframe`s might fail to load, as in most case it's considered a bad practice to load an `iframe` from an untrusted source.

**Comment by marseye** [comment]
Ok. But even W3 Schools continues to declare "The easiest way to play videos in HTML, is to use YouTube" (and most people would still agree with that, though considering Youtube Evil...), even describing only the `<iframe>` method of insertion.

Any workaround ? Since Youtube has to be a trusted source, from the point of view of the videos when they're handpicked by us (and that they're not our ones). Youtube itself is making every effort to provide embarking methods for most social media , but Plume isn't there yet (will it ever be?).

**Comment by fdb-hiroshima** [comment]
I agree, even if I don't like Google, it's still the most used way to share videos. I will update the recommended configuration of the facing web-server (nginx) to allow iframe integration, from any https site as one might also want to showcase something from peertube for example. That said, it's all that can be done from Plume's side, Plume can't control the reverse proxy in front of it, neither can it control an user's browser. As it's the only thing that can be done, I will close this issue. If you find something more we can do, feel free to reopen it.

**Comment by marseye** [comment]
It justed works great since the last update ;) 

Thank you  again ;)
